src/markdown-pages/add-ons/rook.md

```diff
@@ -91,6 +91,11 @@ spec:
   storageClassName: rook-cephfs
 ```
 
+## Network Connectivity
```

Comment:
can we format this similar to weave? we can expand that section with storage requirements eventually https://kurl.sh/docs/add-ons/weave#system-requirements

Reply:
Done. Can you please take a look ?